css/style.css

```diff
@@ -54,7 +54,13 @@ video {
 .main-nav__logo {
 	position: absolute;
 	top: 11px;
-	left: 20px;
+  left: 20px;
+  transition: opacity .2s ease-in;
+  opacity: 0;
+}
+
+.main-nav--colored .main-nav__logo{
+ opacity: 1;
 }
 
 .main-nav__logo__image {
```

Comment:
I think the class name is incorrect, a better approach could be `main-nav__logo-image`

Reply:
Arley didn't add or modify this line... He should not fix it.